### PR TITLE
Add button to enable/disable storage pool

### PR DESCRIPTION
**Pull request by GabrielBrascher**
### Description: 

CloudStack API does support to enable/disable Storage pools via command [updateStoragePool](https://cloudstack.apache.org/api/apidocs-4.14/apis/updateStoragePool.html); however, neither the _legacy UI_ nor CloudStack Primate add such action button.

With that said, this PR adds a button to call the  [updateStoragePool](https://cloudstack.apache.org/api/apidocs-4.14/apis/updateStoragePool.html) API command with the `enabled` parameter set to **true** (for enabling) or **false** (Disabling).

#### Storage enabled
- added _pause-circle_ button
image

- Pop-up so ADMIN can cancel or proceed with operation
image

#### Storage disabled
- added _play-circle_ button
image

- Pop-up so ADMIN can cancel or proceed with operation
image

### CODE REVIEW

**Comment by GabrielBrascher** [comment]
@blueorangutan package

File: src/config/section/infra/primaryStorages.js [hunk 1]
```diff
@@ -63,6 +63,24 @@ export default {
...
       args: ['name', 'tags', 'capacitybytes', 'capacityiops']
     },
+    {
+      api: 'updateStoragePool',
+      icon: 'pause-circle',
+      label: 'label.disable.storage',
+      message: 'message.confirm.disable.storage',
+      dataView: true,
+      defaultArgs: { enabled: false },
+      show: (record) => { return ['Up', 'Connecting', 'Down', 'ErrorInMaintenance'].includes(record.state) }
```
Comment (davidjumani) [inline src/config/section/infra/primaryStorages.js:73]: @GabrielBrascher Does the backend allow disabling a pool if its state is not `Up` ?
https://github.com/apache/cloudstack/blob/master/server/src/main/java/com/cloud/storage/StorageManagerImpl.java#L782

Comment (GabrielBrascher) [reply src/config/section/infra/primaryStorages.js:73]: I think that it makes sence to disable on other cases as well. But currently the backend supports **only** if the pool is Up, I will update this on a couple of minutes.

**Comment by GabrielBrascher** [comment]
@blueorangutan package

**Comment by GabrielBrascher** [comment]
Thanks for the review @davidjumani! Code has been updated.

**Review by davidjumani — APPROVED** [review]

**Pull request merged**